libs/utils/utils.js

```diff
@@ -224,6 +224,7 @@ export const [setConfig, updateConfig, getConfig] = (() => {
       config.base = config.miloLibs || config.codeRoot;
       config.locale = pathname ? getLocale(conf.locales, pathname) : getLocale(conf.locales);
       config.autoBlocks = conf.autoBlocks ? [...AUTO_BLOCKS, ...conf.autoBlocks] : AUTO_BLOCKS;
+      config.susiOptions = conf.susiOptions || {};
```

Comment:
I think the name should be more descriptive, maybe `signInContext`

Reply:
Got it!